### ISSUE TITLE
Change to more distinct colors in memory

### ISSUE THREAD

**Issue by tomaswic** [smab/playhouse-web#83]
Thought I had discovered a new bug but it turned out that it was only the disks having almost exactly the same color.

**Comment by FireyFly** [comment]
Indeed. Also, they should be configurable through the configuration interface.
